helix-view/src/document.rs

```diff
@@ -671,9 +671,13 @@ impl Document {
         &mut self,
         language_id: &str,
         config_loader: Arc<syntax::Loader>,
-    ) {
+    ) -> bool {
```

Comment:
I think it makes sense for this function to return a Result with the context that the language ID is invalid here rather than in `language`.